xclim/ensembles.py

```diff
@@ -721,8 +721,9 @@ def plot_rsqprofile(fig_data):
     Examples
     --------
     >>> from xclim.ensembles import kmeans_reduce_ensemble, plot_rsqprofile
+    >>> from xclim.testing import open_dataset
     >>> is_matplotlib_installed()
-    >>> crit = xr.open_dataset(path_to_ensemble_file).data
+    >>> crit = open_dataset(path_to_ensemble_file).data
```

Comment:
Is this no longer working?

Reply:
For some reason, it is not... I am currently digging. `xr` seems to be refering to xarray everywhere in `ensembles.py` doctest, but not in the other ones.